dtfs-central-api/api-tests/v1/utilisation-reports/post-upload-utilisation-report.api-test.ts

```diff
@@ -127,13 +127,37 @@ describe(`POST ${getUrl()}`, () => {
     expect(azureFileInfo).toHaveLength(1);
   });
 
-  it('saves the report when there is a large number of fee records', async () => {
+  it('saves the report when there is a large number of fee records all for the same facility', async () => {
     // Arrange
     const numberOfReportDataEntriesToCreate = 500;
 
+    const facilityId = '12345678';
+    const reportData: UtilisationReportRawCsvData[] = [];
+    while (reportData.length < numberOfReportDataEntriesToCreate) {
```

Comment:
could this be in a test helper

Reply:
I don't think there is a reason to have a test helper for a single array population (YAGNI)

Reply:
I think that this still should be in a test helper but can be done later, resolving